### DISCUSSION TITLE
Huawei HarmonyOs

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#1948] [Q&A]
Hello all.

As you know last year Huawei devices start to come with new mobile system HarmonyOs. I dont have chance to try model viewer and augmented reality on thesee devices. and I know HarmonyOs have AR Engine too. Is it working on HarmonyOs too ? or do I have to add specific codes or scripts for it. 

Anyone tried on HarmonyOs for model viewer's Augmented Reality ? 

Thanks...

**Comment by elalish** [comment]
I have no idea, but I would also be interested in the results. I would expect the 3D mode to work fine, but AR is a bit more questionable.
